content/community/links.html

```diff
@@ -22,20 +20,17 @@
 <li><a href="http://www.besly.de/" target=_blank>BeSly</a>: German BeOS, Haiku and ZETA knowledge base. Tutorials, translations 
 and more in several languages.</li>
 <li><a href="http://betips.net/" target=_blank>BeTips</a>: The Haiku/BeOS Tip Server.</li>
-<li><a href="https://github.com/HaikuArchives" target=_blank>HaikuArchives</a>: The GitHub repository for many Haiku native open source  projects. <a href="https://haikuarchives.github.io/software.html">Software Archives <a></li>
+<li><a href="https://github.com/HaikuArchives" target=_blank>HaikuArchives</a>: The GitHub repository for many Haiku native open source projects.</li>
+<li><a href="https://haikuarchives.github.io/software.html" target=_blank>HaikuArchives Index</a>: A partial index of the Haiku Archives.</li>
 <li><a href="http://haiku-gazette.blogspot.com" target=_blank>Haiku Gazette</a>: Haiku news in German.</li>
 <li><a href="http://www.haiku-os.it" target=_blank>Haiku OS Italia</a>: News for Italian Haiku users.</li>
```

Comment:
This should be removed, the original website is down and what's there now is used for SEO hacks for an online casino.

Reply:
The one I commented on, http://www.haiku-os.it

Reply:
Removed.  I can see it from my side, but I have a Pi-Hole setup that may be blocking it.